packages/kokkos/algorithms/unit_tests/CMakeLists.txt

```diff
@@ -164,7 +164,7 @@ endif()
 # when compiling for Intel's Xe-HP GPUs.
 if(NOT (KOKKOS_ENABLE_OPENMPTARGET AND KOKKOS_CXX_COMPILER_ID STREQUAL IntelLLVM))
   KOKKOS_ADD_EXECUTABLE_AND_TEST(
-    UnitTest_RandomAndSort
+    AlgorithmsUnitTest_RandomAndSort
```

Comment:
This changes the name of the test (missing underscore)

Reply:
(not a blocker just saying)

Reply:
Given that subpackages are being removed, some of the tests (like `UnitTest`) had conflicting names because the prefixes became the same `Kokkos_`.  The approach I took was to change from `Kokkos<Subpkg>_<remainder>` to `Kokkos_<Subpkg><remainder>`.  But if you would instead prefer changing this to `Kokkos_<Subpkg>_<remainder>`, we can do that as well.  Just let me know how you would like to names these tests consistently.

Reply:
Current version is good enough